src/components/CommentList.jsx

```diff
@@ -0,0 +1,47 @@
+import { useState, useEffect } from "react";
+import { fetchComments, fetchArticleByID } from "../utils/api.js";
+import { useParams } from "react-router-dom";
+
+export default function ListOfComments() {
+  const [allComments, setAllComments] = useState([]);
+  const [articleTitle, setArticleTitle] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const { article_id } = useParams();
+
+  useEffect(() => {
+    fetchComments(article_id).then(({ comments }) => {
+      setAllComments(comments);
+      setIsLoading(false);
+    }) 
+    fetchArticleByID(article_id).then(({ article }) => {
+      console.log(article);
+        setArticleTitle(article.title);
+    });
+  }, [article_id]);
+
+
+  if (isLoading) return <p>Loading, One moment please...</p>;
+  return (
+    <div>
+      <h3 className="ArticleTitleMessage">All Comments for: </h3>
+      <h2 className="ArticleTitleComments" >{articleTitle}</h2>
+      <h2>
+
+      {allComments.map((comment) => { 
+          return (
+            <div>

```

Comment:
pop a `key` attribute in this `<div>` element to get rid of that pesky console error